_episodes/03-sql-joins.md

```diff
@@ -21,6 +21,17 @@ keypoints:
 To combine data from two tables we use the SQL `JOIN` command, which comes after
 the `FROM` command.
 
+Earlier, we talked about relational databases.
```

Comment:
I think I would remove this, it's not clear what the relation is with the next paragraph (are you hinting at normal forms?)